### PR TITLE
Set line color feedback in CSS

### PR DESCRIPTION
**Pull request by loichuder**
https://github.com/silx-kit/h5web/pull/1190#pullrequestreview-1103713685

### CODE REVIEW

File: packages/lib/src/vis/hooks.ts [hunk 1]
```diff
@@ -75,7 +74,7 @@ export function useCameraState<T>(
 }
 
 export function useCustomColors(
```
Comment (loichuder) [inline packages/lib/src/vis/hooks.ts:76]: The hook could be renamed to something else: `useColorCssVariable` ?

File: packages/lib/src/vis/line/LineVis.module.css [hunk 2]
```diff
@@ -4,6 +4,15 @@
...
   margin: 1rem;
+  --curve-color: var(--h5w-line--color, darkblue);
+  --aux-colors: var(
+    --h5w-line--colorAux,
+    orangered,
+    forestgreen,
+    red,
+    mediumorchid,
+    olive
+  );
```
Comment (loichuder) [inline packages/lib/src/vis/line/LineVis.module.css:15]: Following the suggestion in https://github.com/silx-kit/h5web/pull/1190#discussion_r962925905

File: packages/lib/src/vis/hooks.ts (hunk 1 above)
Comment (axelboc) [reply packages/lib/src/vis/hooks.ts:76]: `useCssColors`?

**Review by axelboc — APPROVED** [review]
Nice! Glad it works :+1:

**Pull request merged**